package.json

```diff
@@ -94,6 +99,7 @@
   "lint-staged": {
     "*.{ts,tsx,js,jsx}": [
       "npm run pretty",
+      "npm run lint",
```

Comment:
CI tests make sure lint errors do not pass. Sometimes people (like me) forget to check linting errors :sweat_smile: . Thus, it seems like a great idea to make this check as a precommit hook. :100:

Reply:
cc: @timmyichen

Reply:
YESSSSSSSSSSSSSSSSSSSSSS

Reply:
🎉 nice, thanks!

Reply:
Actually, on second thought, should we bake this into CI instead of making it a pre commit hook? I can imagine a scenario where you'd want to make some dev commits and it could be annoying to have to lint unfinished code in between each commit

Reply:
Sounds great! Will remove it. I also found there is a pre-push hook. So, It looks before push the code should be linted anyhow, can put it there? @timmyichen